app/frontend/stylesheets/config/_colors.scss

```diff
@@ -0,0 +1,10 @@
+// Define variables for your color scheme
+
+// For example:
+$red: #FD1015;
+$blue: #0D6EFD;
+$yellow: #FFC65A;
+$orange: #E67E22;
+$green: #1EDD88;
+$gray: #0E0000;
+$light-gray: #F4F4F4;
```

Comment:
Colours are already being defined inside the config/_variables.scss file.
Can you move any colour you need inside of there and delete this file please ?

Reply:
Yes but not all of them. Only the one you actually use. There's no point in having 20 variables we never use in the app